templates/generic/object_home_content.html

```diff
@@ -35,9 +36,37 @@ <h1 class="{{ title_class|default:"pg-title" }}">{{ title }}</h1>
              hx-indicator=".htmx-indicator">
       <i class="mx-2 htmx-indicator fa fa-spinner fa-pulse"></i>
     </div>
+
+  {% elif enable_search and toggle_archived %}
+    <div class="my-2 flex gap-x-2">
+      <div class="my-2" div>
+        <input class="input input-bordered join-item" type="search"
+               name="{{ search_param_name|default:"search" }}" placeholder="Search..."
+               hx-get="{{ table_url }}"
+               hx-trigger="keyup[this.value.length === 0 || this.value.length > 2] changed delay:500ms, search"
+               hx-target="div.table-container"
+               hx-indicator=".htmx-indicator">
+        <i class="mx-2 htmx-indicator fa fa-spinner fa-pulse"></i>
+      </div>
+      <div class="my-2" div>
+        <label for="show_archived" class="label cursor-pointer flex items-center space-x-2">
+          <span class="label-text mr-2">Show Archived</span>
+          <input
+            type="checkbox"
+            name="show_archived"
+            class="toggle"
+            {% if show_archived == "true" %}checked{% endif %}
+            hx-get="{{ table_url }}{% querystring %}"
```

Comment:
does this work with the search input?

Reply:
https://github.com/dimagi/open-chat-studio/pull/922/commits/dec229f83ad303ceea27972aed6508c3190e38a7

Reply:
even better https://github.com/dimagi/open-chat-studio/pull/922#:~:text=%E2%80%A6-,aeb8a1c,-Merge%20state and updated the loom video